Security/Authentication/Token/JWTUserToken.php

```diff
@@ -37,7 +37,7 @@ public function __construct(array $roles = [], UserInterface $user = null, $rawT
         $this->setRawToken($rawToken);
         $this->setAuthenticated(true);
 
-        $this->providerKey = $providerKey;
+        $this->providerKey = $firewallName;
```

Comment:
Why don't you change the property name to `$this->firewallName` as well?

Reply:
Because the property is protected, so it would have been a BC break